benchmark/benchmark-sysbench-v5.3.0-vs-v5.2.2.md

```diff
@@ -0,0 +1,202 @@
+   user        haproxy                     # Same with the UID parameter.
+   group       haproxy                     # Same with the GID parameter. A dedicated user group is recommended.
+   nbproc      64                          # The number of processes created when going daemon. When starting multiple processes to forward requests, ensure that the value is large enough so that HAProxy does not block processes.
+   daemon                                  # Makes the process fork into background. It is equivalent to the command line "-D" argument. It can be disabled by the command line "-db" argument.
+
+defaults                                   # Default configuration.
+   log global                              # Inherits the settings of the global configuration.
+   retries 2                               # The maximum number of retries to connect to an upstream server. If the number of connection attempts exceeds the value, the backend server is considered unavailable.
+   timeout connect  2s                     # The maximum time to wait for a connection attempt to a backend server to succeed. It should be set to a shorter time if the server is located on the same LAN as HAProxy.
+   timeout client 30000s                   # The maximum inactivity time on the client side.
+   timeout server 30000s                   # The maximum inactivity time on the server side.
+
+listen tidb-cluster                        # Database load balancing.
+   bind 0.0.0.0:3390                       # The Floating IP address and listening port.
+   mode tcp                                # HAProxy uses layer 4, the transport layer.
+   balance roundrobin                      # The server with the fewest connections receives the connection. "leastconn" is recommended where long sessions are expected, such as LDAP, SQL and TSE, rather than protocols using short sessions, such as HTTP. The algorithm is dynamic, which means that server weights might be adjusted on the fly for slow starts for instance.
+   server tidb-1 10.9.18.229:4000 check inter 2000 rise 2 fall 3       # Detects port 4000 at a frequency of once every 2000 milliseconds. If it is detected as successful twice, the server is considered available; if it is detected as failed three times, the server is considered unavailable.
+   server tidb-2 10.9.39.208:4000 check inter 2000 rise 2 fall 3
+   server tidb-3 10.9.64.166:4000 check inter 2000 rise 2 fall 3
+```
+
+## Test plan
+
+1. Deploy TiDB v5.3.0 and v5.2.2 using TiUP.
+2. Use Sysbench to import 16 tables, each table with 10 million rows of data.
+3. Execute the `analyze table` statement on each table.
+4. Back up the data used for restore before different concurrency tests, which ensures data consistency for each test.
+5. Start the Sysbench client to perform the `point_select`, `read_write`, `update_index`, and `update_non_index` tests. Perform stress tests on TiDB via HAProxy. For each concurrency under each load, the test takes 20 minutes.
```

Comment:
Instead of "load", it will be nice to use "workload". "For each concurrency under each workload"

Reply:
```suggestion
5. Start the Sysbench client to perform the `point_select`, `read_write`, `update_index`, and `update_non_index` tests. Perform stress tests on TiDB via HAProxy. For each concurrency under each workload, the test takes 20 minutes.
```